pkg/autodiscovery/providers/kube_services_test.go

```diff
@@ -32,11 +32,16 @@ func TestParseKubeServiceAnnotations(t *testing.T) {
 		{
 			service: &v1.Service{
 				ObjectMeta: metav1.ObjectMeta{
-					UID: types.UID("test"),
+					UID:       types.UID("test"),
+					Namespace: "default",
+					Name:      "myendpoint",
 					Annotations: map[string]string{
-						"ad.datadoghq.com/service.check_names":  "[\"http_check\"]",
```

Comment:
It's better to add new cases than change existing ones, this allows to ensure we don't introduce regressions.
I'd also like to see a broader range of test cases:
  - only svc (already covered)
  - only ep
  - both valid
  - invalid svc annotation, valid ep annotation, we expect the valid one to come through

Reply:
totally agree 👍